src/elements/MetricsSelector/MetricsSelector.js

```diff
@@ -0,0 +1,269 @@
+/*
+Copyright 2019 Iguazio Systems Ltd.
+
+Licensed under the Apache License, Version 2.0 (the "License") with
+an addition restriction as set forth herein. You may not use this
+file except in compliance with the License. You may obtain a copy of
+the License at http://www.apache.org/licenses/LICENSE-2.0.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+implied. See the License for the specific language governing
+permissions and limitations under the License.
+
+In addition, you may not use the software for any purposes that are
+illegal under applicable law, and the grant of the foregoing license
+under the Apache 2.0 license is conditioned upon your compliance with
+such restriction.
+*/
+import React, { useMemo, useRef, useState, useEffect, useCallback } from 'react'
+import PropTypes from 'prop-types'
+import { FieldArray } from 'react-final-form-arrays'
+import classNames from 'classnames'
+import { debounce, isEmpty } from 'lodash'
+import { Form } from 'react-final-form'
+import arrayMutators from 'final-form-arrays'
+import { createForm } from 'final-form'
+
+import { PopUpDialog, FormInput } from 'igz-controls/components'
+import { SelectOption } from 'igz-controls/elements'
+import { TextTooltipTemplate, Tooltip } from 'iguazio.dashboard-react-controls/dist/components'
+import { ReactComponent as Caret } from 'igz-controls/images/dropdown.svg'
+import {
+  filterMetrics,
+  generateMetricsItemsByApplication,
+  getMetricColorById
+} from './metricsSelector.utils'
+import { METRICS_SELECTOR_OPTIONS } from '../../types'
+import Accordion from '../../common/Accordion/Accordion'
```

Comment:
split components imports and utils imports

Reply:
+